tgui/packages/tgui/interfaces/ExaminePanel.js

```diff
@@ -22,8 +24,32 @@ export const ExaminePanel = (props, context) => {
       <Window.Content>
         <Stack fill>
           <Stack.Item width="30%">
-            <Section fill title="Character Preview">
-              {!obscured
+            {headshot.length !== 0 ? (
```

Comment:
What happens when byond passes tgui a null here

Reply:
It blue screens, but the default is "", which is length = 0 and works fine

Reply:
You could probably just do `!!headshot` instead, and that'd be null-safe, I'm pretty sure.

Reply:
Yep

Reply:
> It blue screens, but the default is "", which is length = 0 and works fine

Never trust byond to send what you expect it to send.

Reply:
done